pandas/core/generic.py

```diff
@@ -9104,6 +9103,13 @@ def where(
         3    3
         4    4
         dtype: int64
+        >>> s.mask(s > 1, 10)
```

Comment:
can you add a blank line, maybe worth adding comment that mask and where are inverse selections

Reply:
I chose not to add a blank line because the example above is the corresponding `s.where(s > 1, 10)` example